app/controllers/no_js_controller.rb

```diff
@@ -3,6 +3,7 @@ class NoJsController < ApplicationController
 
   def index
     session[SESSION_KEY] = true
+    analytics.no_js_detect_stylesheet_loaded(location: params[:location])
```

Comment:
Do we think it would speed tings up at all to skip writing to asession? Or I guess the session will get written back no matter what anyways

Reply:
The session detail is used in `JavaScriptRequiredComponent` to display a success banner after the user re-enables JavaScript.

https://github.com/18F/identity-idp/blob/154eaeed3341e91543907a8a8cd9041efcbc0618/app/components/javascript_required_component.rb#L22-L24

But yeah, technically it's not strictly needed for the sign-in flow. I'd be up for micro-optimizing it if we thought we might expect a fair bit of traffic, but I'm hoping the volume will be pretty low overall?

Reply:
yes, I remember why we added it. And yes, seems fine to not worry about optimizing it just yet